packages/rest-typings/src/v1/omnichannel.ts

```diff
@@ -3008,7 +3008,21 @@ export type OmnichannelEndpoints = {
 			department: ILivechatDepartment | null;
 			agents?: ILivechatDepartmentAgents[];
 		};
-		PUT: (params: { department: Partial<ILivechatDepartment>[]; agents: any[] }) => {
+		PUT: (params: {
+			department: {
+				enabled: boolean;
+				name: string;
+				description?: string | undefined;
+				showOnRegistration: boolean;
+				email: string;
+				showOnOfflineForm: boolean;
+				requestTagBeforeClosingChat?: boolean | undefined;
+				chatClosingTags?: string[] | undefined;
+				fallbackForwardDepartment?: string | undefined;
+				departmentsAllowedToForward?: string[] | undefined;
+			};
```

Comment:
i think this is pretty useful in other parts of the code, maybe it should became a type

btw I'm missing the validator

Reply:
Good one, forgot to centralize it. 

On the validator, yeah 😬 the endpoint has some "custom" behavior of adding validators to the custom fields that I haven't looked at how to do with ajv without the "additionalProps": true, cause that allows everything 🤔 so, that's why there's no validator for now